app/pages/clinicworkspace/ClinicPatients.js

```diff
@@ -2558,7 +2558,7 @@ export const ClinicPatients = (props) => {
     const averageGlucose = summary?.bgmStats?.periods?.[activeSummaryPeriod]?.averageGlucose;
     let averageDailyRecords = Math.round(summary?.bgmStats?.periods?.[activeSummaryPeriod]?.averageDailyRecords);
     const averageDailyRecordsUnits = averageDailyRecords > 1 ? 'readings/day' : 'reading/day';
-    if (averageDailyRecords === 0) averageDailyRecords = '>1';
+    if (averageDailyRecords === 0) averageDailyRecords = '<1';
```

Comment:
I think the unit test for this needs to be updated

Reply:
Good catch.  Updated.